content/sensu-core/2.0/getting-started/glossary.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Glossary of Terms"
+linkTitle: "Glossary"
+description: "Introduction to Sensu terms and concepts"
+weight: 10
+version: "2.0"
+product: "Sensu Core"
+menu:
+  sensu-core-2.0:
+    parent: getting-started
+---
+
+#### Agent
+A lightweight client that runs on the infrastructure components you want to monitor. Agents self-register with the backend, send keepalive messages, and execute monitoring checks. Each agent belongs to one or more subscriptions that determine which checks the agent runs. An agent can run checks on the entity it’s installed on or by connecting to a remote proxy entity. [Read more.][1]
+
+#### Backend
+A flexible, scalable monitoring event pipeline. The backend processes event data using filters, mutators, and handlers. It maintains configuration files, stores recent event data, and schedules monitoring checks. You can interact with the backend using the API, command line, and dashboard interfaces. [Read more.][2]
+
+#### Check
+A script run by the agent to determine the state of a system component or collect metrics. Checks specify an interval for execution, one or more subscriptions, and one or more handlers that will process the resulting event data. Once run, a check produces a check result that includes the output of the command and an exit status code. [Read more.][3]
```

Comment:
So just a general thing from a 1.x perspective: 1.x has check definitions and the check executable that those definitions run, which as a whole is usually just called a "check". This definition appears to be talking a little about both as it starts out saying a check is "A script", but then it has other Sensu terms talked about. 

Entirely possible that is the right way to think of things in 2.x, but it is definitely something I like to distinguish in Sensu 1.x as it helps with the learning process.

Reply:
Would this be better?

>A script run by the agent to determine the state of a system component or collect metrics. Check definitions specify the location of the script, an interval for execution, one or more subscriptions, and one or more handlers that will process the resulting event data. Once run, a check produces a check result that includes the output of the command and an exit status code.

Reply:
I think I am being difficult here, or something. Or taking words too literally. This is possibly entirely fine the way it is.

My overall idea here was to ensure we distinguish between a check (the thing defines what command a check runs, the interval, etc.) and a check plugin (the thing that gets ran). Calling a check (I read that as a check definition a script, to me, isn't right in 1.x; it might be valid in 2.x.

Let's just leave it alone unless someone else feels the same way.